src/main/java/Commands.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Commands {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+    public static int numberOfTasks = 0;
+    public static int completedTasks = 0;
+    public static int remainingTasks;
+    public static final Scanner scan = new Scanner(System.in);
+
+    /**
+     * Prints the respective lines or statements based on the commands input by the user.
+     * Returns the boolean value of the command.
+     *
+     * @return boolean value
+     */
+    public static boolean getCommandInput() {
+        String userInput = scan.nextLine();
+        String[] commandInput = userInput.split(" ");
+        switch (commandInput[0]) {
+        case "list":
+            PrintLine.printLine(PrintList.printListDescription(numberOfTasks, completedTasks)
+                    + PrintList.printListItems(tasks, numberOfTasks));
+            break;
+        case "todo":
+            numberOfTasks++;
```

Comment:
Try using class level members to store the number of tasks and the remaining number of tasks.
Do refer to the textbook: https://nus-cs2113-ay2021s1.github.io/website/book/cppToJava/classes/classLevelMembers/
Perks of doing this, you can increase the number of tasks within the constructor and the remaining task can be computed when you make a task as done.

Reply:
Thank you for telling me this. I have refactored the variables into class level members in Task.java.